iotclient/IoTDevice.java

```diff
@@ -34,24 +34,30 @@ public static void main(String[] args) {
         addCliShutdownHook();
         sc = new Scanner(System.in);
         // Check arguments
-        if (args.length < 3) {
+        if (args.length < 6) {
             System.out.println(
-                    "Error: not enough args!\nUsage: IoTDevice <serverAddress> <dev-id> <user-id>\n");
+                    "Error: not enough args!\nUsage: IoTDevice <serverAddress> <truststore> <keystore> <passwordkeystore> <dev-id> <user-id>\n");//<serverAddress> <dev-id> <user-id>\n");
             System.exit(1);
         }
         String serverAddress = args[0];
-        devid = args[1];
-        userid = args[2];
-
-        // Ask for pswd
-        System.out.println("Password for " + userid + ":");
-        password = sc.nextLine();
+        String truststore = args[1];
+        String keystore = args[2];
+        String psw_keystore = args[3];
+        devid = args[4];
+        userid = args[5];
+
+        System.setProperty("javax.net.ssl.trustStore", truststore);
+        System.setProperty("javax.net.ssl.trustStoreType", "JCEKS");
+        System.setProperty("javax.net.ssl.keyStore", keystore);
+        System.setProperty("javax.net.ssl.keyStorePassword", psw_keystore);
+        System.setProperty("javax.net.ssl.keyStoreType", "JCEKS");
+        
 
         // Connection & Authentication
         if (connect(serverAddress)) {
-            userAuth(userid, password);
-            deviceAuth(devid);
-            testDevice();
+            //userAuth(userid, password);
```

Comment:
Removing this part of code breaks commands, because the server is still coded to receive them at the moment. either leave them in or fix them in a branch